### ISSUE TITLE
`models` defined twice in JS client

### ISSUE THREAD

**Issue by nikolasburk** [prisma/prisma#3677]
Generate the JS client for the following datamodel:

```
type User {
  id: ID! @unique
  name: String!
}
```

The generated `index.js` will look as follows:

```js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var prisma_lib_1 = require("prisma-client-lib");
var typeDefs = require("./prisma-schema").typeDefs;

var models = [
  {
    name: "User",
    embedded: false
  }
];
exports.Prisma = prisma_lib_1.makePrismaClientClass({
  typeDefs,
  models,
  endpoint: `https://eu1.prisma.sh/nikolas/testasdasf/dev`
});
exports.prisma = new exports.Prisma();
var models = [
  {
    name: "User",
    embedded: false
  }
];
```

**Comment by Bitti09** [comment]
Don't know if i'm right but for me it looks like this line is the reason:

https://github.com/prisma/prisma/blob/master/client/src/codegen/javascript-client.ts#L25
${this.renderModels()} gets called twice: Line 22 &Line 25

Line 22 was added with this commit:
https://github.com/prisma/prisma/commit/106525f07506cefcfda497727d4ecff5192c8016

**Comment by divyenduz** [comment]
This is resolved in the latest stable (1.23.x) `npm install -g prisma`
